README.md

```diff
@@ -287,103 +264,163 @@ in your package and update the path definitions accordingly. Specifications are
 
 **NOTE:** To ensure your service icons look beautiful on retina-ready displays,
 please supply 2x versions of all icons. No changes are needed to
-`package.json` - simply supply an additional icon file with the text `@2x` in
+`resource.json` - simply supply an additional icon file with the text `@2x` in
 the name before the file extension.
 For example, the icon `icon-cassandra-small.png` would have a retina-ready
 alternate image named `icon-cassandra-small@2x.png`.
 
-### Versioning
+##### CLI Resources
+|Packaging Version|   |
+|-----------------|---|
+|2.0|not supported|
+|3.0|optional|
 
-The registry specification is versioned separately in the file `/repo/meta/version.json`.
+The new `.cli` property allows for a package to configure "native" CLI sub commands for several platforms and
```

Comment:
I'll remove quotes and let you do the larger update. I don't want to mess it up.